packages/sui-js/test/arraySpec.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-env mocha */
+import {expect} from 'chai'
+import sinon from 'sinon'
+
+import {shuffle} from '../src/array/index.js'
+
+describe('@s-ui/js', () => {
+  describe('array:shuffleArray', () => {
+    beforeEach(() => {
+      sinon.stub(Math, 'random').returns(0.5)
+    })
+
+    afterEach(() => {
+      sinon.restore()
+    })
+
+    it('should shuffle an array', () => {
+      const params = [1, 2, 3, 4, 5]
+      expect(shuffle(params)).to.eql([1, 4, 2, 5, 3])
```

Comment:
.to.include.members(

Reply:
https://www.chaijs.com/api/bdd/#method_include

Reply:
https://www.chaijs.com/api/bdd/#method_members

Reply:
shuffle result is always the same?

Reply:
@andresin87 puedes explicar un poco mas porque usar `members` y no `eql` ?

Reply:
> shuffle result is always the same?

doesn't look like it 😉 
https://github.com/SUI-Components/sui/pull/1442/commits/74171acc05cbc339fc9a0c19bc0ba5bd2c23416f

Reply:
yep, the math random is stubbed. sorry